interview-questions.md

```diff
@@ -7,90 +7,97 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would you use the method `super()`?
 
-  Your answer:
+  Your answer: You would use super() when using class inheritance and you want to access the parent class and pull in all variables from the class.
 
-  Researched answer:
+  Researched answer: The super() method refers to a parent class. It is used to call the constructor of a parent class and to access those properties and methods that belong to that parent class. If you use "extends" on a class you must use super() to access the properties in methods in the parent class or else you will receive a reference error.
 
 
 
 2. What is the DOM? What is the virtual DOM?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: The DOM stands for Document Object Model. It is how React can change the output on an application without reloading the entire page. The virtual DOM is what the changes are first logged to, it then gets updated virtually (in the background), and then the DOM is updated to display the change.
```

Comment:
You've pretty much got the right idea here. The Virtual DOM is what is specific to React, while the DOM is universal across the web.